### PR TITLE
CI setup

### LINKED ISSUE #62: Implement CI

### PR DESCRIPTION
**Pull request by gritfessel**
Closes #62

### CODE REVIEW

File: packages/ChangesReloaded-Tests.package/CRTest.class/instance/tearDown.st [hunk 1]
```diff
@@ -0,0 +1,4 @@
+running
+tearDown
+
+	self changesBrowser: nil
```
Comment (marcelgarus) [inline packages/ChangesReloaded-Tests.package/CRTest.class/instance/tearDown.st:4]: As discussed in person, this doesn't do anything.

Also, `setUp` calls `super setUp`. Do we also need to call `super tearDown`? (I'm not sure about this one and, of course, this only applies if we would need this method at all.)

File: packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/changesBrowser..st [hunk 2]
```diff
@@ -0,0 +1,3 @@
+accessing
+changesBrowser: aCRBrowser
+	changesBrowser := aCRBrowser
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/changesBrowser..st:3]:
```suggestion
	
	changesBrowser := aCRBrowser
```

File: packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/changesBrowser.st [hunk 3]
```diff
@@ -0,0 +1,3 @@
+accessing
+changesBrowser
+	^ changesBrowser
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/changesBrowser.st:3]:
```suggestion

	^ changesBrowser
```

File: packages/ChangesReloaded-Core.package/CRBrowser.class/class/open.st [hunk 4]
```diff
@@ -1,4 +1,4 @@
 instance-creation
 open
 
-	ToolBuilder open: self new
\ No newline at end of file
+	^ ToolBuilder open: self new
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/class/open.st:4]: Looks like the linter (?) expects a new line?
Appears multiple times in this Branch

File: packages/BaselineOfChangesReloaded.package/BaselineOfChangesReloaded.class/instance/baseline..st [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+baseline: spec
+	<baseline>
+	spec for: #common do: [
+		spec 
+			package: 'ChangesReloaded-Core';
+			package: 'ChangesReloaded-Tests' with: [ spec requires: #('ChangesReloaded-Core') ].
+			
+		spec
+			group: 'default' with: #('ChangesReloaded-Core');
+			group: 'Tests' with: #('ChangesReloaded-Tests')].
```
Comment (SilvanVerhoeven) [inline packages/BaselineOfChangesReloaded.package/BaselineOfChangesReloaded.class/instance/baseline..st:11]:
```suggestion
			group: 'Tests' with: #('ChangesReloaded-Tests')]
```
Sadly, according to our code stlye guidelines. I'd like to keep to dot.

File: packages/BaselineOfChangesReloaded.package/BaselineOfChangesReloaded.class/instance/baseline..st [hunk 6]
```diff
@@ -0,0 +1,11 @@
+baseline
+baseline: spec
+	<baseline>
```
Comment (SilvanVerhoeven) [inline packages/BaselineOfChangesReloaded.package/BaselineOfChangesReloaded.class/instance/baseline..st:3]:
```suggestion

	<baseline>
```

**Pull request closed without merging**